product_variant_configurator/i18n/es.po

```diff
@@ -204,16 +209,22 @@ msgstr "No crear variantes automáticamente"
 msgid "Expense Account"
 msgstr "Cuenta de Gastos"
 
+#. module: product_variant_configurator
```

Comment:
All these modifications in the PO file don't make sense, as this module is not touching that fields.